workitem/workitem_repository.go

```diff
@@ -110,14 +111,54 @@ func (r *GormWorkItemRepository) Load(ctx context.Context, spaceID uuid.UUID, wi
 	return ConvertWorkItemStorageToModel(wiType, wiStorage)
 }
 
+// LookupIDByNamedSpaceAndNumber returns the work item's ID for the given owner name, space name and item number
+// returns NotFoundError, ConversionError or InternalError
+func (r *GormWorkItemRepository) LookupIDByNamedSpaceAndNumber(ctx context.Context, ownerName, spaceName string, wiNumber int) (*uuid.UUID, *uuid.UUID, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "LookupIDByNamedSpaceAndNumber"}, time.Now())
+	log.Debug(nil, map[string]interface{}{
+		"wi_number":  wiNumber,
+		"space_name": spaceName,
+		"owner_name": ownerName,
+	}, "Loading work item")
+	query := fmt.Sprintf("select wi.id, wi.space_id from %[1]s wi "+
+		"join %[2]s s on wi.space_id = s.id "+
+		"join %[3]s i on s.owner_id = i.id "+
+		"where i.username = ? and s.name = ? and wi.number = ?", "work_items", "spaces", "identities")
```

Comment:
Please use [`space.Space{}.TableName()`](https://github.com/fabric8-services/fabric8-wit/blob/28e6b04108dae248c07b321df44d41bf7bcbdf7d/space/space.go#L94) and [`account.Identity{}.TableName()`](https://github.com/fabric8-services/fabric8-wit/blob/28e6b04108dae248c07b321df44d41bf7bcbdf7d/account/identity.go#L81) instead of those hard-coded `"spaces"` and `"identities"` strings.